src/libraries/System.Data.Common/src/System/Data/DataColumnCollection.cs

```diff
@@ -723,7 +723,7 @@ internal int IndexOfCaseInsensitive(string name)
         {
             int hashcode = _table.GetSpecialHashCode(name);
             int cachedI = -1;
-            DataColumn? column = null;
+            DataColumn? column;
```

Comment:
This can be declared in the loop where it's assigned.

Reply:
Done